async-http-client-backend/cats/src/main/scala/sttp/client/asynchttpclient/cats/AsyncHttpClientCatsBackend.scala

```diff
@@ -22,7 +22,7 @@ import cats.implicits._
 
 import scala.language.higherKinds
 
-class AsyncHttpClientCatsBackend[F[_]: Async: ContextShift] private (
+class AsyncHttpClientCatsBackend[F[+_]: Async: ContextShift] private (
```

Comment:
Is adding variance here somehow related to the original issue? What kind of problems does it solve?